src/app/shared/components/filters-list/working-hours/working-hours.component.html

```diff
@@ -6,14 +6,16 @@
       {{day.value}}</div>
   </div>
   <p class="filter-text">Години роботи</p>
-  <mat-form-field>
+  <mat-form-field> 
     <div class="wrapper" fxLayout='row' fxLayoutAlign='start' fxLayoutGap='1rem'>
       <p class="filter-text">Від</p>
       <input class="time-input" type="text" matInput [ngxMatTimepicker]="timepickerStart" format="24"
-        [formControl]="startTimeFormControl">
+        [formControl]="startTimeFormControl" [max]="getMaxTime()">
```

Comment:
to use functions in data binding is bad approach and can cause poor performance, please use variable instead